model-governance/model_governance_graph.ipynb

```diff
@@ -0,0 +1,520 @@
+    "\n",
+    "You can create the Neptune cluster from [this](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning.html) CloudFormation template\n",
+    "\n",
+    "Also, the instructions to install the Graph notebook jupyter extension to visualize the graph from [HERE](https://github.com/aws/graph-notebook/#gremlin-server) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "social-submission",
+   "metadata": {},
+   "source": [
+    "## Setup Neptune IAM\n",
+    "Follow the instructions here: https://docs.aws.amazon.com/neptune/latest/userguide/iam-auth-enable.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "standing-survivor",
+   "metadata": {},
+   "source": [
+    "## Change Graph config and point to Neptune Endpoint\n",
+    "\n",
+    "If you get an error message: \"UsageError: Cell magic `%%graph_notebook_config` not found.\" use Jupyter Notebooks instead of jupyterlab and make sure that you have already installed the Graph extension from [HERE](https://github.com/aws/graph-notebook/#gremlin-server)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "medieval-canberra",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%graph_notebook_config\n",
+    "{\n",
+    "  \"host\": \"localhost\", ##<Change this value to point to Neptune Cluster.\n",
+    "  \"port\": 8182,\n",
+    "  \"auth_mode\": \"DEFAULT\",\n",
+    "  \"iam_credentials_provider_type\": \"ROLE\",\n",
+    "  \"load_from_s3_arn\": \"\",\n",
+    "  \"ssl\": false,\n",
+    "  \"aws_region\": \"us-east-1\"\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dying-cooling",
+   "metadata": {},
+   "source": [
+    "## Bulk load data into Neptune"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "significant-sigma",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%graph_notebook_version"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "civic-mitchell",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%graph_notebook_config"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "social-consciousness",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%status"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "separate-powell",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "    \n",
+    "curl -X POST \\\n",
+    "    -H 'Content-Type: application/json' \\\n",
+    "    https://<Neptune Cluster Endpoint>:8182/loader -d '\n",
+    "    {\n",
+    "      \"source\" : \"s3://<<S3 Bucket Path to CSV data>>/\", #Change the bucket name/path with the location of uploaded files\n",
+    "      \"format\" : \"csv\",\n",
+    "      \"iamRoleArn\" : \"arn:aws:iam::<<Account ID>>:role/NeptuneLoadFromS3\",\n",
+    "      \"region\" : \"us-east-2\"\n",
```

Comment:
@icywang86rui No, I wouldn’t worry about the region argument here. The users of the notebook needs to change and replace these variable with their own after creating their Neptune cluster

Reply:
Happy to add a comment there if it will make it clearer

Reply:
My main concern is this will cause failures when we run the notebook automatically. Ideally the notebooks should be able to run in all regions in an automated fashion. My team has been trying to incorporate CI into this repo and the goal is to make all the notebook runnable without manual editing.

Reply:
I removed the region value to a placeholder and moved the dependent cell code to markdown